Centaurus.Test.Utils/FakeAlphaWebSocketConnection.cs

```diff
@@ -0,0 +1,26 @@
+﻿using Centaurus.Models;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Centaurus.Test
+{
+    public class FakeAlphaWebSocketConnection : AlphaWebSocketConnection
+    {
+        public FakeAlphaWebSocketConnection()
+            :base(new FakeWebSocket())
+        {
+
+        }
+
+        public void SetState(ConnectionState state)
```

Comment:
How about exposing those methods in the original `AlphaWebSocketConnection` class?
I don't see the need in the fake class that fakes no logic at all, and used only to setup protected fields of the original class. It's also possible to set fields directly using reflection (more hacky way).

The same for `FakeAuditorWebSocketConnection`